testsuite/features/profiles/github_runner/Docker/authprofile/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Container used to test Content Management feature
+#
+# VERSION               1.0.0
+
+FROM localhost:5001/cucutest/systemsmanagement/uyuni/master/docker/containers/uyuni-master-testsuite
+MAINTAINER Michael Calmer "mc@suse.com"
```

Comment:
Do we really want him as maintainer here?

Reply:
fixed at a new commit. thanks.